### DISCUSSION TITLE
Leave insert mode smoothly

### DISCUSSION THREAD

**Discussion by MiohitoKiri5474** [LazyVim/LazyVim#2352] [Q&A] [answered]
I'm using starter version and now I'm working some configuration.
When I leave insert mode with ESC then press `j` or `k` immediately, the line I just edited will swap with previous line or next line (depends on which key I press) unless I wait a second.
Is any configuration I can modify to avoid it? This feature interfere me a lot.

**Comment by dpetka2001** [comment]
You're inside tmux?

**Reply by MiohitoKiri5474** [reply]
Yes I'm inside tmux, does it matter?

**Reply by MiohitoKiri5474** [reply, marked as answer]
ohh yes it **does** matters, I add `set -s escape-time 0` into my tmux config and it works.
Thanks a lot!

**Reply by dpetka2001** [reply]
Yes, it's a known bug affecting Neovim when inside tmux.
